src/test/java/simpaths/model/PersonTest.java

```diff
@@ -0,0 +1,99 @@
+package simpaths.model;
+
+import org.junit.jupiter.api.*;
+import simpaths.data.Parameters;
+import simpaths.model.enums.Country;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("Person methods in absence of a Model")
+public class PersonTest {
+
+    Person testPerson = new Person(true);
```

Comment:
Minor point, but it'd be preferable to create the person in a @BeforeAll method, so the same object isn't shared across multiple tests. It still needs to be declared here (`Person testPerson;`) so it's accessible across methods, but I would put `testPerson = new Person(true);` inside `setupParams` (and probably rename it `setup` - a single setup method that does multiple things is fine).

Reply:
I'll refactor that and test - I think I had played around with placing it and ended up combining the declaration and construction out of frustration 😁